pkg/controller/nodes/output_resolver.go

```diff
@@ -54,11 +54,19 @@ func (r remoteFileOutputResolver) ExtractOutput(ctx context.Context, nl executor
 		actualVar = variable
 	}
 
+	var output *core.Literal
```

Comment:
maybe add some comments here to make this more readable? For example:
```
// retrieving task output
...
// resolving binding attribute paths (if exist)
```
I found this much more difficult to follow than I would have expected at first look.

Reply:
added